modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceCurrencies.testcase

```diff
@@ -14,6 +14,100 @@ definition {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "This is a test for COMMERCE-11393. Can update currency format for Indian Rupee"
+	@priority = 3
+	test CanUpdateCurrencyFormatForINR {
+		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false";
+		property portal.acceptance = "false";
+
+		task ("Given a Minium site with a user without a role is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			JSONUser.addUser(
+				userEmailAddress = "testuser@liferay.com",
+				userFirstName = "test",
+				userLastName = "user",
+				userScreenName = "testuser");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "testuser@liferay.com");
+
+			JSONUser.addUserToSite(
+				groupName = "Minium",
+				userEmailAddress = "testuser@liferay.com");
+		}
+
+		task ("When the format pattern for Indian Rupee currency is updated to '₹ ##,##,##0.00'") {
+			CommerceNavigator.goToCurrenciesAdmin();
+
+			CommerceNavigator.gotoEntry(entryName = "Indian Rupee");
+
+			Type(
+				key_fieldLabel = "Format Pattern",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = "₹ ##,##,##0.00");
+
+			Button.clickSave();
+		}
+
+		task ("And a simple product with base price '1000000' is created") {
+			CommerceJSONProductsAPI._addCommerceProduct(
+				catalogName = "Minium",
+				productName = "Simple Product",
+				productType = "Simple");
+
+			CommerceProducts.openProductsAdmin();
+
+			CommerceProducts.searchProduct(search = "Simple");
+
+			CommerceNavigator.gotoEntry(entryName = "Simple Product");
+
+			CommerceEntry.gotoMenuTab(menuTab = "SKUs");
+
+			CommerceNavigator.gotoEntry(entryName = "Simple Product_default");
+
+			SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+			Type(
+				key_fieldLabel = "Base Price",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = 1000000);
+
+			Button.clickPublish();
+		}
+
+		task ("And Minium Portal channel currency is updated to 'Indian Rupee'") {
+			CommerceChannels.openChannelsAdmin();
+
+			CommerceNavigator.gotoEntry(entryName = "Minium Portal");
+
+			Select(
+				key_fieldLabel = "Currency",
+				locator1 = "Select#GENERIC_SELECT_FIELD",
+				value1 = "Indian Rupee");
+
+			Button.clickSave();
+		}
+
+		task ("And the user logs in") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "testuser@liferay.com");
+		}
+
+		task ("Then the Simple Product's price is changed to '₹ 8,17,20,000.00' applied to the currency format") {
+			AssertTextEquals(
+				key_productName = "Simple Product",
+				locator1 = "CommerceAccelerators#PRODUCT_CARD_PRICE",
+				value1 = "₹ 8,17,20,000.00");
+		}
```

Comment:
Can you add a couple more assertions here?
e.g. the pattern is applied in all instances where the price is shown: product details, mini-cart, order pages, etc.